xtable-core/src/main/java/org/apache/xtable/avro/AvroSchemaConverter.java

```diff
@@ -435,6 +440,11 @@ private Schema fromInternalSchema(InternalSchema internalSchema, String currentP
             Schema.createFixed(
                 internalSchema.getName(), internalSchema.getComment(), null, fixedSize),
             internalSchema);
+      case UUID:
+        Schema uuidSchema =
+            Schema.createFixed(internalSchema.getName(), internalSchema.getComment(), null, 16);
+        uuidSchema.addProp("logicalType", "uuid");
```

Comment:
> This is going to be confusing to someone without knowledge of XTable being involved since `logicalType` is an existing concept and there is a logical type with a string base type in avro. We should make it prefixed with xtable or something like that so someone looking at the schema in Hudi commit is aware that this is some special context

Thanks for the feedback! Already replace the prop with "xtableLogicalType" in the latest commit!=